.gitignore

```diff
@@ -5,6 +5,7 @@ node_modules
 .idea
 # Visual Studio Code
 .vscode
+.devcontainer
```

Comment:
devcontainers can be useful for the development. Why ignoring them?

Reply:
I assumed the project didn't have devcontainers intentionally. But I've removed from ignore and committed mine.

https://github.com/nearform/openapi-transformer-toolkit/pull/318/commits/aa91880ccfc9fd524c00be5b4f1a002e61b75ef9